kustomize/overlay/7.1.0/securebanking/bohocode/kustomization.yaml

```diff
@@ -1,6 +1,10 @@
 namespace: bohocode
 commonLabels:
   app.kubernetes.io/name: "forgerock"
+  app: ig
+  component: ig
+  vendor: forgerock
+  tier: middle
```

Comment:
what does tier mean  / do?

Reply:
They are labels, if you look at admin ui pod there is a `tier: ui` 
So looks like it's just a way of stating what role they play, guessing middle is because ig routes to different services?